Lesson5_HW/shop/productsapp/views.py

```diff
@@ -0,0 +1,37 @@
+from django.http import JsonResponse
+from django.template.loader import render_to_string
+from django.views.generic import ListView
+
+from .forms import ProductForm
+from .models import ProductModel
+
+
+class ProductsList(ListView):
+    model = ProductModel
+    fields = '__all__'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['title'] = 'Просмотр товаров'
+        return context
+
+
+def ProductCreate(request):
```

Comment:
По PEP-8 именование функций должно быть выполнено в snake_case
```suggestion
def product_create(request):
```